src/app/store/index.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  ActionReducer,
+  ActionReducerMap,
+  createFeatureSelector,
+  createSelector,
+  MetaReducer
+} from '@ngrx/store';
+import { environment } from '../../environments/environment';
+import * as fromAuthorization from './reducers/authorization.reducer';
+import * as fromCourses from './reducers/courses.reducer';
+
+
+export interface AppState {
+  [fromAuthorization.authorizationFeatureKey]: fromAuthorization.AuthorizationState;
+  [fromCourses.coursesFeatureKey]: fromCourses.CoursesState;
+}
+
+export const reducers: ActionReducerMap<AppState> = {
+  // [fromAuthorization.authorizationFeatureKey]: fromAuthorization.reducer,
```

Comment:
Why did we comment out this ?

Reply:
This code was autogenerated from schematics, for some reason the compiler complains about the type (something like the key must be a string) so I decided to keep it just for future reference.